deploy/crds/kieapp.crd.yaml

```diff
@@ -1871,6 +1871,41 @@ spec:
                                 to generated options in JAVA_OPTS. e.g. '-Dsome.property=foo'
                               type: string
                           type: object
+                        kafkaExt:
+                          description: KafkaExt kafka configuration to be used by
+                            the KieApp
+                          properties:
+                            acks:
+                              description: The number of acknowledgments the producer
+                                requires the leader to have received before considering
+                                a request complete.
+                              type: string
+                            autocreateTopics:
+                              description: Allow automatic topic creation.
+                              type: string
+                            bootstrapServers:
+                              description: A comma separated list of bootstrapServers
+                              type: string
+                            clientID:
+                              description: Identifier to pass to the server when making
+                                requests
+                              type: string
+                            extDisabled:
+                              description: Disable the Kafka Extension
+                              type: string
```

Comment:
Is this needed? We users should disabled configured kafka? If Kafka integration is not configured, there is no need to disabled it.

Reply:
I guess, it should be exposed only on the ui, if enabled expose the other kafka parameters just like we have for AMQ configuration.

Reply:
By default is disabled